forecast_tools.R

```diff
@@ -86,8 +86,8 @@ plot_data = function(data) {
     text = getURL(
       "https://raw.githubusercontent.com/weecology/PortalData/master/Rodents/moon_dates.csv"))
   target_moon=unique(data$NewMoonNumber)
-  period_code = dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
-    dplyr::select(Period)
+  period_code = as.integer(dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
+    dplyr::select(Period))
```

Comment:
now the error is "updates were rejected because a pushed branch tip is behind its remote counterpart. check out this branch and integrate the remote changes before pushing again"